appengine/accessible/js/blocks.js

```diff
@@ -19,23 +19,20 @@
  */
 
 /**
- * @fileoverview Music blocks for Blockly.
+ * @fileoverview Blocks for the Blockly Games music demo.
  * @author sll@google.com (Sean Lip)
  */
 'use strict';
 
-goog.provide('Blockly.Blocks.music');
-
-goog.require('Blockly.Blocks');
+goog.provide('Music.Blocks');
 
+goog.require('Blockly');
+goog.require('Blockly.JavaScript');
 
 var MUSIC_DUMMY_TOOLTIP = 'Dummy tooltip';
 var MUSIC_DUMMY_HELPURL = 'Dummy help URL';
 
-/**
- * Common HSV hue for all blocks in this category.
- */
-Blockly.Blocks.music.HUE = 20;
```

Comment:
I noticed that, in the other games (turtle/maze etc.), the individual block colors were hardcoded per-block, so I thought it best to be consistent. What would you prefer? I'm fine either way.